data/family/FreeBSD.yaml

```diff
@@ -0,0 +1,7 @@
+---
+grafana::cfg_location: '/usr/local/etc/grafana.ini'
+grafana::data_dir: '/var/db/grafana',
```

Comment:
the comment at the end isn't valid. If you remove it, the tests might pass.

Reply:
Oops, my bad. Thanks!